build/build.sh

```diff
@@ -60,3 +60,13 @@ if [[ $(uname -s) == 'Linux' ]]; then
 fi
 
 popd
+
+# Use a naive substring searching to check if coverage is requested.
+if [[ $@ == *'--config=coverage'* ]]; then
+    # generate coverage for Java
+    pushd source
+    # Set PATH for Java tests
+    PATH=$PATH:`pwd`/bazel-bin/neuropod/multiprocess/
+    bazel coverage --collect_code_coverage --instrumentation_filter='/java[/:]' --combined_report=lcov --coverage_report_generator=@bazel_tools//tools/test/CoverageOutputGenerator/java/com/google/devtools/coverageoutputgenerator:Main //neuropod/bindings/java/...
+    popd
+fi
```

Comment:
Can we move this to `build/test.sh` and `build/test_gpu.sh`? We don't want the build script to be running tests

Reply:
Also, we want to add a `--nocache_test_results` here to make sure that bazel actually is running all the tests

Reply:
Maybe move it to build/coverage.h ? In test, I would need to add some flag to not run "bazel coverage" when user only want to run tests.

Reply:
I will test if nocache_test_results works in this case